Modules/_pickle.c

```diff
@@ -2054,6 +2055,43 @@ save_float(PicklerObject *self, PyObject *obj)
 }
 
 static int
+_Pickler_write_large_bytes(
+    PicklerObject *self, const char *header, Py_ssize_t header_size,
+    PyObject *payload, Py_ssize_t payload_size)
+{
+    assert(self->output_buffer != NULL);
+    assert(self->write != NULL);
+    PyObject *result;
+
+    // Commit the previous frame
+    if (_Pickler_CommitFrame(self, 0))
```

Comment:
New PEP 7 rule requires braces even around the `if` body containing a simple `return` statement. The code in this module is older than this rule, but the new code should obey it.